bluesky/callbacks/broker.py

```diff
@@ -15,21 +16,28 @@ class LiveImage(CallbackBase):
     field : string
         name of data field in an Event
```

Comment:
Can I just put **kwargs, and note that they are all kwargs to mpl imshow? (note that I will need to update the cross section viewer to take all the mpl stuff